### PR TITLE
Data storage, reading, writing functionalities

### PR DESCRIPTION
**Pull request by ParthGandhiNUS**
Added data storage, reading, and writing functionalities.
Currently the text file only stores the students' names.

### CODE REVIEW

File: src/main/java/classify/datacommands/DataHandler.java [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+//@@author ParthGandhiNUS
 public class DataHandler {
+    private static final String ANALYSING_INPUT_MESSAGE = "Analysing Inputs...";
```
Comment (Cryolian) [inline src/main/java/classify/datacommands/DataHandler.java:18]: Good use of constants

File: src/main/java/classify/datacommands/DataStorage.java [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+    
+    //@@author ParthGandhiNUS
+    /**
+     * Accesses the parent directory to try to make the parent directory.
+     * Tries to write the lines which are made by the writeStudentInfo Method
+     * 
+     * @param lines Arraylist containing the processed current tasks in the masterStudentList
+     */
+    public static void writeStudentInfoFile(List <String> lines){
+        try {
```
Comment (Cryolian) [inline src/main/java/classify/datacommands/DataStorage.java:33]: good use of exceptions

**Comment by Cryolian** [comment]
LGTM

File: src/main/java/classify/datacommands/DataStorage.java [hunk 3]
```diff
@@ -2,4 +2,20 @@
 
 public class DataStorage {
 
+    /**
+     * Used to create the parent folder for a certain file
+     * 
+     * @param parentPath    Path file containing the path of folder we wanna make the text file in
+     * @throws IOException  Triggers whenever the input for the path or creation of the directory is improper
+     */
+    public static void createParentFileFolder (Path parentPath) throws IOException {
```
Comment (tayponghee) [inline src/main/java/classify/datacommands/DataStorage.java:56]: LGTM

**Pull request merged**